emData/generate_PR.py

```diff
@@ -0,0 +1,103 @@
+# Automatically generates ProjectionRouterTop.h, and ProjectionRouterTop.cc
```

Comment:
@djcranshaw Please add the following imports to ensure Python3 compatibility:
```python
from __future__ import absolute_import, print_function
```
They have to be included before anything else.

Reply:
Thanks, done.